### DISCUSSION TITLE
files left in /upload, some added and some not

### DISCUSSION THREAD

**Discussion by enboig** [immich-app/immich#2411] [Q&A]
I started asking for help at discord, but I think it is better to ask here.

My upload folder is huge:
```
root@servidor7:/mnt/md0/docker/immich/Photos# du -hs *
35G     library
1,7G    thumbs
37G     upload
root@servidor7:/mnt/md0/docker/immich/Photos# 
```

I uploaded lot of files using cli, and now some are left only in /upload and some are left in /upload and in /library (I used a custom script to search for duplicated files):
```
492110806 bytes each:
./Photos/upload/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/eabbedf1-8e40-428d-8b55-2095dbfb52f3.mp4
./Photos/upload/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/2a666b2d-0aa5-487c-ae46-86525c5aa258.mp4
./Photos/upload/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/d38c0d39-a563-4be8-81f6-e7e857edf63d.mp4
./Photos/library/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/2022/2022-01-06/VID_20220106_084654.mp4

...

3955515 bytes each:
./Photos/upload/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/197038a8-7f73-4526-b255-5cdbd81f61a6.jpg
./Photos/upload/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/32b30f13-05bb-4c12-9c8f-720104e4718d.jpg
./Photos/upload/f2a8dbef-64e5-4c78-a874-c6a0bcb2997b/96d6b282-33e7-46db-8d17-17edfa203633.jpg
```
I checked the database (`cat dump.sql | grep 'eabbedf1-8e40-428d-8b55-2095dbfb52f3'`) and there is no reference of files in upload folder (neither the ones which have a copy in /library, neither the others); the file in /library is there.

How should I proceed to ensure no photo is lost (and no space is wasted)?

**Comment by alextran1502** [comment]
You can try to click on storage migration job to see if the files are moved correctly

![image](https://github.com/immich-app/immich/assets/27055614/4b2a8271-5c03-4633-b896-765ea73014f8)

**Reply by enboig** [reply]
Nothing happens

**Comment by enboig** [comment]
My only guess is:
1. move all upload files to another folder
2. upload again using immich-cli

I would loose albums in initial upload, but images should be upload (and repeated ones shouldn't)

**Reply by enboig** [reply]
Just in case, if I set `--delete` can I be sure the images have been uploaded?

**Comment by enboig** [comment]
I moved all files, uploaded again them using cli and all seems correct.
